### PR TITLE
Podcasts and articles

### PR DESCRIPTION
**Pull request by rahulrumalla**

### CODE REVIEW

**Review by ra-phael — APPROVED** [review]
🙏

File: README.md [hunk 1]
```diff
@@ -31,8 +31,15 @@ A curated list of awesome links related to Soulbound Tokens.
...
 
 ## Articles
+- [Souldbound by Vitalik Buterin](https://vitalik.ca/general/2022/01/26/soulbound.html)
+- [Decentralized Society: Finding Web3's Soul](https://papers.ssrn.com/sol3/papers.cfm?abstract_id=4105763)
+- [Addressing the most common misconceptions about Account-bound tokens](https://timdaub.github.io/2022/05/30/addressing-the-most-common-misconceptions-about-account-bound-tokens/)
+- [Souldbound tokens should be signed claims](https://katelynsills.com/blockchain/soulbound-tokens/)
 
 ## Podcasts
+- [16 - Decentralized Society - DeSoc | Glen Weyl, Puja Ohlhaver, Leon Erichsen](https://podcasts.apple.com/us/podcast/16-decentralized-society-desoc-glen-weyl-puja-ohlhaver/id1609313639?i=1000564942612)
+- [Unchained podcast Ep. 360 - How Soulbound Tokens Could Reduce Speculation and Improve DAO Voting](https://unchainedpodcast.com/how-soul-bound-tokens-could-reduce-speculation-and-improve-dao-voting-ep-360/)
```
Comment (ra-phael) [inline README.md]: Should we also write the guests for this one?

Comment (rahulrumalla) [reply README.md]: I just copied the titles as-is. Can update the guests

Comment (ra-phael) [reply README.md]: Great, thank you!

**Pull request merged**